sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringOperations.scala

```diff
@@ -188,6 +189,302 @@ case class EndsWith(left: Expression, right: Expression)
 }
 
 /**
+ * A function that trim the spaces from both ends for the specified string.
+ */
+case class StringTrim(child: Expression)
+  extends UnaryExpression with String2StringExpression {
+
+  def convert(v: UTF8String): UTF8String = v.trim()
+
+  override def toString: String = s"TRIM($child)"
```

Comment:
I thought that also, but maybe what we want is the function name, not the class name, or I can remove the `toString` method directly. What do you think?